Source/Dafny/DafnyAst.cs

```diff
@@ -2126,6 +2130,10 @@ public enum AutoInitInfo { MaybeEmpty, Nonempty, CompilableValue }
         return a.IsNumericBased(NumericPersuasion.Real) ? a : null;
       } else if (a.IsNumericBased()) {
         return a.Equals(b) ? a : null;
+      } else if (a.IsBitVectorType) {
+        var aa = a.AsBitVectorType;
```

Comment:
Can you not do something like `if (a.IsBitVectorType) return Equals(a,b) ? a : null`

Reply:
Done